tests/standalone-vault/variables.tf

```diff
@@ -8,7 +8,12 @@ variable "acm_certificate_arn" {
   description = "The ARN of an existing ACM certificate."
 }
 
-variable "license_file" {
-  type        = string
-  description = "The local path to the Terraform Enterprise license to be provided by CI."
```

Comment:
Same comment about retaining this input.

Reply:
input variable "license_file" is retained and is used to input license file path in fixture module invocation. removed object variable "tfe_license"